libnm/meson.build

```diff
@@ -19,6 +13,11 @@ cflags = [
   '-DNMRUNDIR="@0@"'.format(nm_pkgrundir)
 ]
 
+if have_fake_typelibs
+  deps += [ gir_dep ]
```

Comment:
Square brackets are not necessary when adding a dependency to a list.

Reply:
fixed

Reply:
fixed by commit 98b46537fa1a91941b5f7a8252b3fce85554a855